cmd/tailscaled/tailscaled_windows.go

```diff
@@ -98,26 +121,29 @@ func (service *ipnService) Execute(args []string, r <-chan svc.ChangeRequest, ch
 	}()
 
 	changes <- svc.Status{State: svc.Running, Accepts: svcAccepts}
+	syslogf("Service running")
 
-	for ctx.Err() == nil {
+	for {
 		select {
 		case <-doneCh:
+			return false, windows.NO_ERROR
 		case cmd := <-r:
 			log.Printf("Got Windows Service event: %v", cmdName(cmd.Cmd))
 			switch cmd.Cmd {
 			case svc.Stop:
+				changes <- svc.Status{State: svc.StopPending}
+				syslogf("Service stop pending")
 				cancel()
```

Comment:
cancel() // so BabysitProc will kill child process

(I had to remind myself how that worked)